app/client.py

```diff
@@ -0,0 +1,74 @@
+import requests
+from pprint import pprint
+
+
+class Client:
+    def __init__(self, host):
+        self.host = host
+        self.token = None
+
+    def signup(self, username, password, account_name):
```

Comment:
Add some meaningful logging for every endpoint calling. It would be very helpful for the client users. E.g., for signup: `New account {account_name} has been created by {username}.`